chapter04/Plugins/CUQIpy-CIL/demo_ct.ipynb

```diff
@@ -297,184 +356,2680 @@
BEjuu0X54c25zipdQpphtz6GM+EdYVihmzZsiVVbmtrS5VjNHyxMZ5i4zrFXBs6HtI2xfzWxP52hO6DGGLjbPX0t0JfPoQQQghRKHr5EEIIIUSh6OVDCCGEEIVySmg+YihaJ5HFTxd7fox/MratIrUtGzZsSJUbGhpSZfrWR40alSq7Pmf6wqltoB+dfnpCfQL91y70q4f8qqEYE2zLFy+DuTvo86cPn+fzOP34zDvi+vVpQ8b5oFaF+hL2lWvPpyGgvoR1c+1xnNSMjBw5MlVmPA13/VCLwrZ5nDFJaDf2hfExXDuG4tWE9CdcW8xxw/vI1dlQk8N+cp1y/nm/E9+zqC9zcYWI0Yf05Djx9TWr/s+niwy1HWvj3sbZ0pcPIYQQQhSKXj6EEEIIUSj91u1SUVFR+ryTNbWx71qS52e9PLc7hYi1UdawxVnq4lZbfkrnFlZ+3na3KPLzMW0e2qYd2prLbYZue3RlhLb9hkJ5M8Q1P5W7Y/O5g7pqi/BTOd0PvpD2dC+wLyHXFct0KdAt48Jx0UVDm9O1wfVBm3Pchw4d6rYtuvhCbha6Smgn9tW1Od1BdXV1qfL+/ftTZYaC59qka4Sh3d054jGue4Z2p9t069atqXKeIc2znk986TKyhEcPtZU3sdt8Y8hLuqAvH0IIIYQoFL18CCGEEKJQ9PIhhBBCiELpt5oPlywh02O3KGXdsuojqw6jL32leW61DbF79+5Umf5nbuUjrj+T527bti1Vpp6AvlBqRqqrq1Nl+vHd9rgFkdoU+vx96drNyvUGxBdenToMalcIt6DSb8/5djUDHAdtGNrWy2283A7t87VTd0EbUqPD9bFz585UOTT/rhaGa4E2p46GodupnXD1JGbldnXn0A21blZuM25HZwhz2o0aH4Z+v+CCC7rsh1m5Zod9+9WvfpUq79q1y3xk2XIaG3Y8RJHpMvIMnZA1bUieISN6ir58CCGEEKJQ9PIhhBBCiEKJevl466237G/+5m+srq7OBg0aZBMnTrQvf/nLqc/BSZJYY2Oj1dbW2qBBg2zmzJm2efPm3DsuhBBCiFOTKM3HV77yFfunf/onW7VqlU2ZMsWeffZZ++QnP2lVVVV29913m5nZgw8+aMuWLbOVK1fapEmT7L777rNZs2bZ1q1by/zKPlw/U14pfLs6N6s2Is80yFk0IHm3FSJLKmrGati0aVOqPH369FSZMQ7c+hg7gfENGMeBOgxqBNgW0567fn/qA3gtffjsK8OzUytB3OO0IcuMQcG+hOKbMGS6e31o7VDbENLGUI/AvrhlHouNvRGKxcLz3XFTk9PR0ZEq83go7Dj1SLSTezw0jlAMGc4R1yLn2+0L9UUs857auHFjqsy4PSFinkV9qY2I/Z3Jsy9Z45uw7r6M+9Fbol4+fvGLX9iHPvQhu+mmm8zMbMKECfbv//7v9uyzz5rZHwawfPlyW7p0qc2ZM8fMzFatWmXV1dW2evVqmz9/flmdnZ2dqQcKb2ghhBBCnF5EuV2uu+46+8lPfmIvvviimf1B1fzzn//cbrzxRjMza21ttba2Nps9e3bpmsrKSpsxY4atW7euyzqbm5utqqqq9Gfs2LG9HYsQQgghTgGivnzcc889duTIEZs8ebINGDDA3n77bbv//vvttttuM7P/297Fz9HV1dW2Y8eOLutcsmSJLVq0qFTu6OjQC4gQQghxGhP18vHtb3/bvvWtb9nq1attypQptmHDBlu4cKHV1tbavHnzSud15Svrzr9VWVnZZZrzmNwuPT3WVV0hX1hf6i5i8eWKyTquEHmmfyb0EX/wgx/01udqH+hHpwYg5FenRoC6JObbcDUijNMwevToVJl+dcZHoB6FWgnGR/DF+aC+ILQeQloJXu8ep02pm2EMCrZNbUsoZwbjhLhQTxDKn8K2fHlkzNLrJaRdYl2h3E7UCI0bNy5VdvO18FrG6WDsnDFjxqTKXEs1NTWpMtePO4dcl9Qqce38+te/TpX7Ms9U1tTzRca3yFOrmFXDEfP8z6on7I6ol4+//uu/tnvvvdduvfVWMzObOnWq7dixw5qbm23evHmlBd3W1pYKetPe3l72NUQIIYQQ70yiNB9vvPFG2RvXgAEDSm/ldXV1VlNTYy0tLaXjx44ds7Vr15btYBBCCCHEO5OoLx+33HKL3X///TZu3DibMmWKPf/887Zs2TL71Kc+ZWZ/+PyycOFCa2pqsvr6equvr7empiYbPHiwzZ07t08GIIQQQohTi6iXj3/4h3+wL37xi7ZgwQJrb2+32tpamz9/vv2///f/SucsXrzYjh49agsWLLDDhw/btGnTbM2aNVExPsz+4Dc64TvKO0aFr67Y43n1oydt55kLgNCnHLNPPNaHy/Lvfve7VJnxEhjTgNoJF2oAGFOCvvDt27enyvSlM26ImwuEx0JaB1/sDDOzvXv3psrMHeLqT3g/hWIphHQXbIt+fHcOOL/UADAmBeH1zK/C4+4csl+8luOgXoRrh+uednLHzfmlDTkH1NVQV0GYj8XV0tAm1BtRL8J76pJLLkmVX3rppVSZug7X5rQR8+VQu8K2Y5+xbjmkB4ol9JzsS21bTFuhdXkyc3exLva1p0S9fAwdOtSWL19uy5cv93assbHRGhsbe9UhIYQQQpzeKLeLEEIIIQpFLx9CCCGEKJQot0uRuHE+SJb90Xn77bJcn1WnkafWJVSXz68X8gGG6qb/mr5z6htc33gofwY1Hi+//HKqfNFFF6XKe/bsSZWpEXB964wpwX4zfg01HrQT9Sm+fB60CbUP1CNQK8EYI4Rb41070P/M2CmMIcG2OC5qRhgvw9W6cN1Sf8C6ma4hFKuDY3PnlPMTivPBcRGuXdbnts35pg5q3759qTLXNY+zb1z37hzShtRFnYh4fYIDBw5YFmJiJ4XiX5DQs8nXdt46CxKTVybrb0eeMUxi5stFXz6EEEIIUSh6+RBCCCFEoejlQwghhBCF0m81H
```

Comment:
The posterior mean image with NUTS sampler in experimental looks very different.

Reply:
Updated. It works fine now.